app-text/mupdf/mupdf-1.19.1.recipe

```diff
@@ -0,0 +1,134 @@
+SUMMARY="A lightweight XPS and PDF rendering library"
+DESCRIPTION="The renderer in MuPDF is tailored for high quality anti-aliased \
+graphics. It renders text with metrics and spacing accurate to within \
+fractions of a pixel for the highest fidelity in reproducing the look of a \
+printed page on screen.
+
+MuPDF is also small, fast, and yet complete. It supports PDF 1.7 with \
+transparency, encryption, hyperlinks, annotations, searching and more. It also \
+reads XPS/OpenXPS documents and CBZ (Comic Book archive) files."
+HOMEPAGE="https://www.mupdf.com/"
+COPYRIGHT="2006-2022 Artifex Software, Inc"
+LICENSE="AGPL-3.0"
+REVISION="1"
+SOURCE_URI="https://mupdf.com/downloads/archive/mupdf-$portVersion-source.tar.xz"
+CHECKSUM_SHA256="b5eac663fe74f33c430eda342f655cf41fa73d71610f0884768a856a82e3803e"
+SOURCE_DIR="mupdf-$portVersion-source"
+PATCHES="mupdf-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+# get the library version number (excluding patch rev) from port version
+libVersion=$(echo $portVersion | cut -d"." -f1,2)
```

Comment:
You now get a $libVersion = 1.19, I think this should be 1.19.1 (prior version was 1.8, but $portVersion was also 1.8)

Reply:
Checked some things on my end here with changes the $libVersion to use $portVersion, also adding libVersionCompat:
```diff
- libVersion=$(echo $portVersion | cut -d"." -f1,2)
+ libVersion="$portVersion"
+ libVersionCompat="$libVersion compat >= ${libVersion%.**}"
```
Using compat = 1.19 here, seems there are some compat breakage between prior versions, from where I could check (thanks @extrowerk ) it ony goes up to 1.17.0: https://abi-laboratory.pro/index.php?view=timeline&l=mupdf

Reply:
Hold on on changes atm, last check to import fitz yield a missing library ...

Reply:
> You now get a $libVersion = 1.19, I think this should be 1.19.1 (prior version was 1.8, but $portVersion was also 1.8)

This was my intent but as you say it's better that this is libVersionCompat. With mupdf the API is only compatible within minor versions (1.19, 1.8 etc), at 1.12.0 they introduced patch versions as well which are compatible with each other.

Reply:
> Hold off on changes atm, last check to import fitz yield a missing library ...

I noticed this after I pushed, oops! I think it is because libmupdf.so symlink in system/lib doesn't exist. We should create it (not sure how, I thought I had but the one in the recipe goes in system/develop/lib I think) but also link pymupdf with the versioned file directly

Reply:
Changes you suggested have been pushed. Just need to work out this last kink